docs/multicluster/user-guide.md

```diff
@@ -565,29 +572,32 @@ Pod-to-Pod connectivity**.
 
 ```yaml
 apiVersion: v1
-data:
-  controller_manager_config.yaml: |
-    apiVersion: multicluster.crd.antrea.io/v1alpha1
-    kind: MultiClusterConfig
-    ...
-    podCIDRs:
-      - "10.10.1.1/16"
 kind: ConfigMap
 metadata:
   labels:
     app: antrea
   name: antrea-mc-controller-config
   namespace: kube-system
+data:
+  controller_manager_config.yaml: |
+    apiVersion: multicluster.crd.antrea.io/v1alpha1
+    kind: MultiClusterConfig
+    podCIDRs:
+      - "10.10.1.1/16"
 ```
 
 ```yaml
-antrea-controller.conf: |
```

Comment:
@luolanzone I assume this is wrong as I don't find enablePodToPodConnectivity option in antrea-controller config.

Reply:
You are right, it should be antrea-agent's configuration instead of antrea-controller.